seqr/management/commands/lift_project_to_hg38.py

```diff
@@ -81,7 +82,7 @@ def handle(self, *args, **options):
                 ))
 
         # Lift-over saved variants
-        _update_variant_samples(matched_sample_id_to_sample_record, elasticsearch_index)
+        _update_variant_samples(matched_sample_id_to_sample_record, None, elasticsearch_index)
```

Comment:
the user is only used for logging and since logs for ad-hoc manage commands don't actually go to stackdriver/ anywhere persistent theres no reason to add unnecessary code complexity